pkgs/top-level/fonts.nix

```diff
@@ -0,0 +1,267 @@
+{ pkgs, callPackage, callPackages, buildEnv, lowPrio, recurseIntoAttrs
+, nodePackages, perlPackages, pythonPackages }:
+
+rec {
+
+  andagii = callPackage ../data/fonts/andagii { };
+
+  anonymous-pro = callPackage ../data/fonts/anonymous-pro { };
+
+  arkpandora-ttf = callPackage ../data/fonts/arkpandora { };
+
+  aurulent-sans = callPackage ../data/fonts/aurulent-sans { };
+
+  baekmuk-ttf = callPackage ../data/fonts/baekmuk-ttf { };
+
+  bakoma-ttf = callPackage ../data/fonts/bakoma-ttf { };
+
+  cabin = callPackage ../data/fonts/cabin { };
+
+  caladea = callPackage ../data/fonts/caladea {};
+
+  cantarell = callPackage ../data/fonts/cantarell-fonts { };
+
+  carlito = callPackage ../data/fonts/carlito {};
+
+  comfortaa = callPackage ../data/fonts/comfortaa {};
+
+  comic-neue = callPackage ../data/fonts/comic-neue { };
+
+  comic-relief = callPackage ../data/fonts/comic-relief {};
+
+  corefonts = callPackage ../data/fonts/corefonts { };
+
+  culmus = callPackage ../data/fonts/culmus { };
+
+  clearlyU = callPackage ../data/fonts/clearlyU { };
+
+  cm-unicode = callPackage ../data/fonts/cm-unicode {};
+
+  crimson = callPackage ../data/fonts/crimson {};
+
+  dejavu = lowPrio (callPackage ../data/fonts/dejavu-fonts {
```

Comment:
I'm inclined to agree that allowing different names in this case makes sense. But I would suggest adopting Debian style package name with a `fonts-` prefix rather than a `-fonts` suffix for all font packages, so that alphanumeric sorting clumps them together. That is, `fonts-google`, `fonts-dejavu`, `fonts-font-awesome`, etc.